### ISSUE TITLE
foxy 1.2.0 no more works with PHP < 7.1 and should reflect this in composer.json and version number

### ISSUE THREAD

**Issue by gajdusek** [fxpio/foxy#43]
Thank you for creating this excellent composer plugin!

Commit 7d3d8242b47ab introduced at least one PHP 7.1+ feature, a nullable type, in
```php
public function __construct(
        IOInterface $io,
        Config $config,
        ProcessExecutor $executor,
        Filesystem $fs,
        FallbackInterface $fallback = null,
        ?VersionConverterInterface $versionConverter = null
    ) {
        $this->io = $io;
        ...
   }
```
Despite this breaking change the foxy version number did not bump to 2.0.0 and composer.json still requires only `php >= 5.3.3`:
```json
{
    ...
    "require": {
        "php": ">=5.3.3",
        ...
    },
    ...
```
This is the first error encountered when running composer with PHP < 7.1. Maybe there are more.

Petr

**Comment by francoispluchino** [comment]
I thank you for your feedback. This needs to be fixed.

**Comment by francoispluchino** [comment]
Fixed by c4154031e18ff404f07608c6778c30bd9afe9966.
